aspnetcore/includes/net-core-prereqs-vs-2.2.md

```diff
@@ -1,2 +1,5 @@
-* [Visual Studio 2017 version 15.9 or later](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload
-* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
\ No newline at end of file
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) with the **ASP.NET and web development** workload
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
+
+> [!WARNING]
+> Visual Studio 2017 must target .NET Core SDK 2.2.1nn. Use the **Compatible with Visual Studio 2017** download button on the .NET Core SDK download page. See the GitHub issue [.NET Core SDK 2.2.2nn and 2.1.6nn do not work with Visual Studio 2017](https://github.com/dotnet/sdk/issues/3124) to resolve problems with Visual Studio 2017 and the .NET Core SDK.
```

Comment:
You could simplify this -- 

> If you use Visual Studio 2017, see [link to issue] for information about .NET Core SDK versions that don't work with Visual Studio.

If you include the ".1nn" notation it would be a good idea to include examples, e.g. ".200".  Or ".1x" might work.